start/src/main/java/org/apache/accumulo/start/classloader/AccumuloClassLoader.java

```diff
@@ -86,10 +86,10 @@ public static String getAccumuloProperty(String propertyName, String defaultValu
       return defaultValue;
     }
     try {
-      FileBasedConfigurationBuilder<PropertiesConfiguration> propsBuilder =
-          new FileBasedConfigurationBuilder<>(PropertiesConfiguration.class)
-              .configure(new Parameters().properties().setURL(accumuloConfigUrl));
-      PropertiesConfiguration config = propsBuilder.getConfiguration();
+      var config = new PropertiesConfiguration();
+      try (var reader = new FileReader(accumuloConfigUrl.getFile())) {
+        config.getLayout().load(config, reader);
```

Comment:
I wasn't aware of the `read()` method before. Apparently, we can just call `config.read(reader)` here. I also wasn't aware of `FileHandler` before, which I see is being used when saving files.

I think we should try to be consistent. We should either use:
```java
var config = PropertiesConfiguration();
try (var reader = new FileReader()) {
  config.read(reader);
}
// and
try (var writer = new FileWriter()) {
  config.write(writer);
}
```

OR

```java
var config = new PropertiesConfiguration();
var fileHandler = new FileHandler(config);
fileHandler.load(file);
// and
fileHandler.save(file);
```

Ultimately, they will both do the same thing, but I'm not sure which one is cleaner, since we still need to handle exceptions. Either way, but we should consistently use one pattern or the other.